### ISSUE TITLE
Alt text for overrides should use the alt text from the main image 

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/cgov-digital-platform#972]
Alt text for overrides should use the alt text from the main image, and we should hide the alt text field for the override images.

**Comment by andyvanavery31** [comment]
Revisit the requirements and intention of this ticket to ensure the most descriptive alt text is provided to the users. Alt text for an override can be more specific than the main image, but if no alt text is provided for the override image, the main image alt text needs to be used. Factoids or data can display different information on different overrides, so the alt text for an override can be more descriptive. When we do a fall back from not having an override alt text, the fall back should be the main image. All images should always have alt text.

**Comment by andyvanavery31** [comment]
Is there any other similar features to this that have been implemented since this ticket's creation?
